boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpurad_iron_0_9_0.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: nrf54h20dk/nrf54h20/cpurad/iron
+name: nRF54H20-DK-nRF54H20-Radio (IRONside SE compatible) (revision 0.9.0)
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - xtools
+  - zephyr
+sysbuild: true
+ram: 192 # Guessed value
+flash: 256 # Guessed value
```

Comment:
The RAM is as per spec, and flash corresponds to the memory map we have in the docs, so I would say that the comments here are superfluous

Reply:
fixed